### DISCUSSION TITLE
`\` as `λ` in haskell

### DISCUSSION THREAD

**Discussion by scarf005** [tonsky/FiraCode#1467] [Ideas]
lambdas are written as `\` in haskell. ex) `\x y -> x + y` is same as `lambda x, y = x + y` in python.
it'd be a nice addition for haskell to render it as  `λ`, say `λ x y -> x + y`

**Comment by tonsky** [comment]
You want to replace any backslash to lambda? Aren’t backslashes used for anything else in Haskell?

**Reply by scarf005** [reply]
yeah, found out it'd be impossible to distinguish between  \ with parameter (\x -> x) and [function name starting with \ (\\)
](https://hoogle.haskell.org/?hoogle=/). also `\n` and `\` in strings. it'd impossible to distinguish them within bounds of ligatures.

could you close the discussion as finished/wontfix?
